### DISCUSSION TITLE
[Crash] [ARCropolis 3.4.0] Game randomly crashes on CSS during 8-Player Smash

### DISCUSSION THREAD

**Discussion by DShip91** [Raytwo/ARCropolis#349] [Issues] [answered]
System Firmware Version
14.1.1

Atmosphere Version
1.3.2

Smash Version
13.0.1

ARCropolis Version
3.4.0

Can the crash be consistently reproduced?
Kind of...

At what moment does the crash happen?
On the character select screen while 8-Player Smash is active

Details:
The game will occasionally lock up and then crash on the character selection screen while eight characters are selected. It's entirely random when the game crashes but waiting on the screen seems to increase the likleihood of a crash occurring.

Crash report
[01653097339_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/8746148/01653097339_01006a800016e000.log) (I hope this is right--it's the most recent crash log on my system.)

**Comment by Raytwo** [comment]
[Already reported](https://github.com/Raytwo/ARCropolis/issues/252). Closing this.

**Comment by Raytwo** [comment, marked as answer]
Issue fixed as of ARCropolis 3.5.0 (coming out tonight)
